executor/distsql.go

```diff
@@ -489,9 +489,7 @@ func (e *XSelectIndexExec) nextForSingleRead() (*Row, error) {
 		if err != nil {
 			return nil, errors.Trace(err)
 		}
-		// Use time.UTC instead of session's timezone because coprocessor evaluator has
```

Comment:
This comment is a mistake, since it didn't get expected result here, I add this comment as a remind.
However, I was wrong, it's not because coprocessor evaluator, but the wrong handle of timezone encoding
when ddl add index.

After I correct the encoding, it's behavior is right now: the value get from TiKV is in UTC, pass the
session's timezone to get the result.